src/Akka.Persistence.MongoDb.Hosting/MongoDbJournalOptions.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Text;
+using Akka.Configuration;
+using Akka.Hosting;
+using Akka.Persistence.Hosting;
+
+#nullable enable
+namespace Akka.Persistence.MongoDb.Hosting;
+
+public class MongoDbJournalOptions : JournalOptions
+{
+    private static readonly Config Default = MongoDbPersistence.DefaultConfiguration()
+        .GetConfig(MongoDbJournalSettings.JournalConfigPath);
+
+    public MongoDbJournalOptions() : this(true)
+    {
+    }
+
+    public MongoDbJournalOptions(bool isDefault, string identifier = "mongodb") : base(isDefault)
+    {
+        Identifier = identifier;
+        AutoInitialize = true;
+    }
+
+    /// <summary>
+    /// Connection string used to access the MongoDb, also specifies the database.
+    /// </summary>
+    public string ConnectionString { get; set; } = "";
+
+    /// <summary>
+    /// Name of the collection for the event journal or snapshots
+    /// </summary>
+    public string Collection { get; set; } = "EventJournal";
+
+    /// <summary>
+    /// Name of the collection for the event journal metadata
+    /// </summary>
+    public string MetadataCollection { get; set; } = "Metadata";
+
+    /// <summary>
+    /// Transaction
+    /// </summary>
+    public bool UseWriteTransaction { get; set; }
```

Comment:
What's the default value for this?

Reply:
Explicit default has been set - Confirmed it matches reference conf.

https://github.com/akkadotnet/Akka.Persistence.MongoDB/blob/1f98a99f2865995a1497148592d2dd9e11721535/src/Akka.Persistence.MongoDb/reference.conf#L11